app/view_users/view_users.html

```diff
@@ -12,7 +12,12 @@ <h1 flex>Manage Users</h1>
         <md-button md-colors="::{background: 'primary-900'}"
                    class="md-fab md-secondary button-add-user"
                    aria-label="Add Users"
-                   ng-click="ctrl.showDialog(null)">
+                   ng-click="ctrl.showDialog(
+                    'PanelDialogCtrl',
+                    'view_users/panel_editUser.html',
+                    'user-dialog',
+                    { 'sipClient': null },
+                    ctrl.queryUsers)">
```

Comment:
Should we extract this to a function in the controller?
There is code duplication.